app/src/main/java/com/T05/krowdtrialz/ui/subscribed/ScanActivity.java

```diff
@@ -72,9 +74,13 @@ public void onDecoded(@NonNull Result result) {
 
                 if (resultArray.length > 1) { // QRCode
                     db = Database.getInstance();
-                    db.getExperimentByID(resultArray[0], new Database.GetExperimentCallback() {
+                    expRegistration = db.getExperimentByID(resultArray[0], new Database.GetExperimentCallback() {
                         @Override
                         public void onSuccess(Experiment experiment) {
+                            if (resultArray[5] == "None" || resultArray[6] == "None"){
+                                resultArray[5] = null;
+                                resultArray[6] = null;
```

Comment:
What do `5` and `6` signify? Would it be pertinent to give these indeces some definition?

Reply:
We figured it was enough to leave this comment:
`// resultArray has the following format {experimentID, type, Pass Count, Fail Count, Value, Longitude, latitude}`

Reply:
We figured it was enough to leave this comment:
`// resultArray has the following format {experimentID, type, Pass Count, Fail Count, Value, Longitude, latitude}`